### PR TITLE
[WFLY-11112]: WildFly intermittently fails to add pooled-connection-factory but it still got registered after.

### PR DESCRIPTION
**Pull request by ehsavoie**
Adding JNDI aliases in the same step as the JNDI main name when adding a PooledConnectionFactory.

Jira: https://issues.jboss.org/browse/WFLY-11112

### CODE REVIEW

File: connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java [hunk 1]
```diff
@@ -921,4 +921,11 @@
     @Message(id = 117, value = "%s is not a valid %s implementation")
     OperationFailedException notAValidDataSourceClass(String clz, String dsClz);
 
+    @LogMessage(level = INFO)
+    @Message(id = 118, value = "Binding connection factory named %s to alias %s")
```
Comment (jmesnil) [inline connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java]: I'm not sure but I think this log is also used when binding Admin Objects.

**Review by jmesnil — CHANGES_REQUESTED** [review]
Looks good from the messaging perspective but the connector log might require to be rephrased.

@maeste Could you please review it?

**Review by maeste — APPROVED** [review]

**Comment by ehsavoie** [comment]
retest this please

File: connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java (hunk 1 above)
Comment (ehsavoie) [reply connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java]: not the same log entry

**Review by jmesnil — APPROVED** [review]

**Pull request merged**